lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -105,7 +101,6 @@ export default function BasicLtiLaunchApp({ rpcServer }) {
   const fetchGroups = useCallback(async () => {
```

Comment:
fetchGroups no longer is part of fetchCount. I'm hesitate to throw incFetchCount and decFetchCount away even through its only used for the fetchContentUrl.  There is still value in those functions for cleaning up error so I'd like to keep it.